packages/cli/src/UserManagement/routes/users.ts

```diff
@@ -363,6 +365,16 @@ export function usersNamespace(this: N8nApp): void {
 				user_id: invitee.id,
 			});
 
+			await this.externalHooks.run('user.profile.update', [
+				invitee.email,
+				{
+					firstName,
+					lastName,
+					email: invitee.email,
+				},
```

Comment:
I think providing the sanitized invitee or updated user would allow for more options for the hooks creator and less redundancy, wdyt?

Reply:
done